dev-tools/jenkins_ci.sh

```diff
@@ -56,6 +56,10 @@ fi
 # Remove any old builds
 rm -rf ../builds
 
+# Disassociate from reference repo
+git repack -a -d
+rm -rf ../.git/objects/info/alternates
```

Comment:
@dimitris-athanasiou This might be a bit safer by doing something like:

```bash
readonly GIT_TOPLEVEL=$(git rev-parse --show-toplevel 2> /dev/null)

rm -f "${GIT_TOPLEVEL}/.git/objects/info/alternates"
```

Reply:
Done! Thanks for the suggestion!